include/dt-bindings/pwm/pwm.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2019 Vestas Wind Systems A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_PWM_PWM_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_PWM_PWM_H_
+
+/**
+ * @name PWM polarity flags
+ * The `PWM_POL_*` flags are used with `pwm_pin_set_cycles`,
+ * `pwm_pin_set_usec`, or `pwm_pin_set_nsec` to specify the polarity
+ * of a PWM pin.
+ * @{
+ */
+/** PWM pin normal polarity (active-high pulse). */
+#define PWM_POL_NORMAL	(0 << 0)
+
+/** PWM pin inverted polarity (active-low pulse). */
+#define PWM_POL_INV	(1 << 0)
+
+/** @cond INTERNAL_HIDDEN */
+#define PWM_POL_MASK	0x1
```

Comment:
The naming of these defines was chosen to mimic the currently used GPIO defines in `include/dt-bindings/gpio/gpio.h`. As pointed out on the API meeting, the naming scheme should be changed to match the flags naming scheme of the `topic-gpio` branch (e.g. `PWM_POLARITY_NORMAL` instead of the abbreviated `PWM_POL_NORMAL`).